samples: ipc: ipc-service: Added NRF7002DK Support

HerkeDekker opened the pull request:
Added board overlay files, updated the Kconfig to support the nRF7002DK, and revised the documentation accordingly.

NordicBuilder commented:
Thank you for your contribution!
It seems you are not a member of the nrfconnect GitHub organization. External contributions are handled as follows:
Large contributions, affecting multiple subsystems for example, may be rejected if they are complex, may introduce regressions due to lack of test coverage, or if they are not consistent with the architecture of nRF Connect SDK.
PRs will be run in our continuous integration (CI) test system.
If CI passes, PRs will be tagged for review and merged on successful completion of review. You may be asked to make some modifications to your contribution during review.
If CI fails, PRs may be rejected or may be tagged for review and rework.
PRs that become outdated due to other changes in the repository may be rejected or rework requested.
External contributions will be prioritized for review based on the relevance to current development efforts in nRF Connect SDK.  Bug fix PRs will be prioritized.
You may raise issues or ask for help from our Technical Support team by visiting https://devzone.nordicsemi.com/.

*Note: This comment is automatically posted and updated by the Contribs GitHub Action.*

peknis commented on samples/ipc/ipc_service/README.rst:
```diff
@@ -95,6 +100,38 @@ Use these overlays when building the IPC sample to test the following scenarios:
      west build -p -b nrf5340dk/nrf5340/cpuapp -T sample.ipc.ipc_service.nrf5340dk_icmsg_cpuapp_sending .
      west build -p -b nrf5340dk/nrf5340/cpuapp -T sample.ipc.ipc_service.nrf5340dk_icmsg_cpunet_sending .
 
+ **nRF7002 DK**
```
@dchat-nordic, we can open a further PR to make these different DKs tabs.

nordicjm commented on samples/ipc/ipc_service/Kconfig.common:
```diff
@@ -7,6 +7,7 @@
 config APP_IPC_SERVICE_SEND_INTERVAL
 	int "Ipc service sending interval [us]"
 	default 30 if BOARD_NRF5340DK_NRF5340_CPUNET
+	default 30 if BOARD_NRF7002DK_NRF5340_CPUNET
```
`	default 30 if BOARD_NRF5340DK_NRF5340_CPUNET || BOARD_NRF7002DK_NRF5340_CPUNET`

nordicjm commented on samples/ipc/ipc_service/Kconfig.sysbuild:
```diff
@@ -10,4 +10,6 @@ config REMOTE_BOARD
 	string "The board used for remote target"
 	default "nrf5340dk/nrf5340/cpunet" if BOARD_NRF5340DK_NRF5340_CPUAPP
 	default "nrf5340dk/nrf5340/cpunet" if BOARD_NRF5340DK_NRF5340_CPUAPP_NS
+	default "nrf7002dk/nrf5340/cpunet" if BOARD_NRF7002DK_NRF5340_CPUAPP
```
`	default "nrf7002dk/nrf5340/cpunet" if BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF7002DK_NRF5340_CPUAPP_NS`

NordicBuilder commented:
This pull request has been marked as stale because it has been open (more than) 30 days with no activity. Remove the stale label or add a comment saying that you would like to have the label removed otherwise this pull request will automatically be closed in 7 days. Note, that you can always re-open a closed pull request at any time.

HerkeDekker commented:
> merge commits need to be removed

Hey JM, whoops, my mistake! I rebased the branch, so that should be fixed now. I also implemented your other requested changes. I wasn’t sure whether those changes were expected of me, but since the PR had stalled, I figured they were :-)

HerkeDekker commented:
> commits need to be squashed without the @ mention of the name, and a commit description needs to be added

Whoops, once again my bad :-). Fixed now!

HerkeDekker commented:
Hey @nordicjm,

I was on vacation for two weeks and just noticed some of the workflows failing due to spacing/trailing issues. I’ve just fixed them, and also added myself as the person who signed off the commit, as this is required too. Could you approve the pull requests to run again? Should be fine now. Thanks in advance.

Kind regards,
Herke Dekker

nordicjm commented:
> Hey @nordicjm, I see that the Compliance workflow is failing, but I’m not sure what’s causing it. Is this something I should address, or is it an issue outside my control?

Nothing needed on your side, will get it merged when someone is available to bypass the check
